### DISCUSSION TITLE
checkbox内部文字一直居右，能否可配置成居左或居右？

### DISCUSSION THREAD

**Discussion by genggengya** [ant-design/ant-design-mobile#5246] [Q&A]
如题，现在不可配置，文字一直在右，想让它在左，但是没有相关api，能否支持一下呢？

O   TEXT
TEXT   O

**Comment by RadiumAg** [comment]
自己改样式就行

**Reply by genggengya** [reply]
改样式确实可以，但是失去了点击文字选中的功能

**Reply by genggengya** [reply]
只能把checkbox变得和父级盒子一样大，定位到上面，在视觉上实现点击文字选中的效果，但是没有官方一个api来的实在

**Reply by RadiumAg** [reply]
> 只能把checkbox变得和父级盒子一样大，定位到上面，在视觉上实现点击文字选中的效果，但是没有官方一个api来的实在

你可以自己再封装一层，来控制你的这个需求[😁](https://apps.timwhitlock.info/emoji/tables/unicode#emoji-modal)

**Reply by genggengya** [reply]
嗯呐

**Comment by awmleer** [comment]
目前不支持的，但是可以自己包一层：

```
- MyWrappedCheckbox （在这里绑定和处理 onClick 事件）
  - Text
  - Checkbox （这里只渲染一个勾选框，不渲染文字）
```

**Reply by genggengya** [reply]
嗯嗯，好嘞，👍🏻
